account_invoice_refund_link/views/account_invoice_view.xml

```diff
@@ -5,82 +5,82 @@
-                        <field name="number" string="Number"/>
-                        <field name="reference" string="Reference"/>
-                        <field name="origin" string="Source"/>
-                        <field name="refund_reason" string="Refund reason"/>
-                        <field name="amount_total" string="Total"/>
-                        <field name="state" string="Status"/>
-                    </tree>
-                </field>
-            </page>
-        </notebook>
-        <xpath expr="//page[@name='other_info']//field[@name='origin']" position="after">
-            <field name="refund_invoice_id"
-                   readonly="1"
-                   attrs="{'invisible': [('refund_invoice_id', '=', False)]}"
-            />
-            <field name="refund_reason"
-                   readonly="1"
-                   attrs="{'invisible': [('refund_reason', '=', False)]}"
-            />
-        </xpath>
-    </field>
-</record>
-
-<record id="view_supplier_account_invoice_add_refunds_details_form" model="ir.ui.view">
-    <field name="name">Account invoice (supplier) | add Refunds details (form)</field>
-    <field name="model">account.invoice</field>
-    <field name="inherit_id" ref="account.invoice_supplier_form"/>
-    <field name="arch" type="xml">
-        <notebook position="inside">
-            <page name="refunds"
-                  string="Refunds"
-                  attrs="{'invisible':[('type', 'not in', ['in_invoice', 'out_invoice'])]}"
-            >
-                <field name="refund_invoice_ids" nolabel="1">
-                    <tree decoration-info="state == 'draft'"
-                          decoration-muted="state == 'cancel'"
-                          string="Refund invoices">
-                        <field name="partner_id" string="Supplier"/>
-                        <field name="date_invoice" string="Bill date"/>
-                        <field name="number" string="Number"/>
-                        <field name="reference" string="Reference"/>
-                        <field name="origin" string="Source"/>
-                        <field name="refund_reason" string="Refund reason"/>
-                        <field name="amount_total" string="Total"/>
-                        <field name="state" string="Status"/>
-                    </tree>
-                </field>
-            </page>
-        </notebook>
-        <xpath expr="//page[@name='other_info']//field[@name='name']" position="after">
-            <field name="refund_invoice_id"
-                   readonly="1"
-                   attrs="{'invisible': [('refund_invoice_id', '=', False)]}"
-            />
-            <field name="refund_reason"
-                   readonly="1"
-                   attrs="{'invisible': [('refund_reason', '=', False)]}"
-            />
-        </xpath>
-    </field>
-</record>
+    <record id="view_customer_account_invoice_add_refunds_details_form" model="ir.ui.view">
```

Comment:
Please don't change this only for esthetic reasons, as it creates a great diff with no need.

Reply:
> Please don't change this only for esthetic reasons, as it creates a great diff with no need.

@pedrobaeza Yes, But it's look ugly, so I changed.

Reply:
Please don't